### PR TITLE
Implement setting locations in expense items

### PR DESCRIPTION
**Pull request by williums**

### CODE REVIEW

File: Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java [hunk 1]
```diff
@@ -174,6 +186,30 @@ public void onClick(View v) {
 		});
 		
 		incomplete = false;
+
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java:189]: Indentation here is all weird, reindent in Eclipse.

**Comment by indragiek** [comment]
Looks fine besides the indentation. Feel free to self merge once that's fixed.

**Comment by williums** [comment]
Oops. I'll fix that right away.

File: Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java (hunk 1 above)
Comment (williums) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java:189]: Sorry, which lines? It looks fine for me in Eclipse

Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java:189]: Look at it on GitHub, Eclipse might not be showing you the mix of tabs/spaces.

Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java:189]: You can just select the code and do Ctrl+Shift+F in Eclipse to indent all of it correctly.

**Pull request merged**